.github/workflows/rust.yml

```diff
@@ -36,10 +36,10 @@ jobs:
     outputs:
       isRust: ${{ steps.diff.outputs.isRust }}
     steps:
-    - uses: actions/checkout@v3
-    - name: Detect Changes
-      uses: './.github/actions/diffs'
-      id: diff
+      - uses: actions/checkout@v3
```

Comment:
Just curious, what is causing the auto formatting?

Reply:
I have a few formatter extensions on vscode but I believe `Prettier` is the one that auto formatted this file.